<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layout editor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1120</width>
      <height>570</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>570</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>[0,0]</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>Add</string>
     </property>
     <addaction name="actionStation"/>
     <addaction name="actionTrack"/>
     <addaction name="actionShunt_track"/>
     <addaction name="actionPlatform"/>
     <addaction name="actionSignal"/>
     <addaction name="actionCrossing"/>
     <addaction name="separator"/>
     <addaction name="actionLabel"/>
     <addaction name="actionPanel"/>
    </widget>
    <addaction name="menuAdd"/>
    <addaction name="actionRemove_element"/>
    <addaction name="actionRemove"/>
    <addaction name="actionTest"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="statusTip">
    <string/>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Clear all</string>
   </property>
  </action>
  <action name="actionStation">
   <property name="text">
    <string>Station</string>
   </property>
  </action>
  <action name="actionTrack">
   <property name="text">
    <string>Track</string>
   </property>
  </action>
  <action name="actionPlatform">
   <property name="text">
    <string>Platform</string>
   </property>
  </action>
  <action name="actionSignal">
   <property name="text">
    <string>Signal</string>
   </property>
  </action>
  <action name="actionCrossing">
   <property name="text">
    <string>Crossing</string>
   </property>
  </action>
  <action name="actionShunt_track">
   <property name="text">
    <string>Shunt track</string>
   </property>
  </action>
  <action name="actionLabel">
   <property name="text">
    <string>Label</string>
   </property>
  </action>
  <action name="actionPanel">
   <property name="text">
    <string>Panel</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionRemove_element">
   <property name="text">
    <string>Remove element</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
